src/sentry/incidents/subscription_processor.py

```diff
@@ -487,11 +491,15 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         ]
         fired_incident_triggers = []
         with transaction.atomic(router.db_for_write(AlertRule)):
+            # Triggers is the threshold - NOT an instance of a trigger
             for trigger in self.triggers:
-                if alert_operator(
+                if alert_operator(  # operator.gt/operator.lt
                     aggregation_value, trigger.alert_threshold
-                ) and not self.check_trigger_status(trigger, TriggerStatus.ACTIVE):
+                ) and not self.check_trigger_matches_status(trigger, TriggerStatus.ACTIVE):
```

Comment:
nit: might be nice to short circuit this check first, that way we don't need to do the `alert_operator` step if the check is already active.

Reply:
Ah this was just a rename of the helper method to be more clear